2021-10-06 23:43:45,532 INFO    MainThread:37814 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 23:43:45,532 INFO    MainThread:37814 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-06 23:43:45,533 INFO    MainThread:37814 [wandb_init.py:_log_setup():357] Logging user logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_234345-1rhpelve/logs/debug.log
2021-10-06 23:43:45,533 INFO    MainThread:37814 [wandb_init.py:_log_setup():358] Logging internal logs to /Users/kiranganeshan/reading-group/RL/wandb/run-20211006_234345-1rhpelve/logs/debug-internal.log
2021-10-06 23:43:45,533 INFO    MainThread:37814 [wandb_init.py:init():390] calling init triggers
2021-10-06 23:43:45,533 INFO    MainThread:37814 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {}
2021-10-06 23:43:45,534 INFO    MainThread:37814 [wandb_init.py:init():435] starting backend
2021-10-06 23:43:45,534 INFO    MainThread:37814 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 23:43:45,550 INFO    MainThread:37814 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 23:43:45,561 INFO    MainThread:37814 [backend.py:ensure_launched():140] started backend process with pid: 37843
2021-10-06 23:43:45,563 INFO    MainThread:37814 [wandb_init.py:init():440] backend started and connected
2021-10-06 23:43:45,574 INFO    MainThread:37814 [wandb_init.py:init():499] updated telemetry
2021-10-06 23:43:45,575 INFO    MainThread:37814 [wandb_init.py:init():522] communicating current version
2021-10-06 23:43:46,224 INFO    MainThread:37814 [wandb_init.py:init():527] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 23:43:46,224 INFO    MainThread:37814 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-06 23:43:46,443 INFO    MainThread:37814 [wandb_init.py:init():562] starting run threads in backend
2021-10-06 23:43:49,606 INFO    MainThread:37814 [wandb_run.py:_console_start():1651] atexit reg
2021-10-06 23:43:49,608 INFO    MainThread:37814 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.REDIRECT
2021-10-06 23:43:49,608 INFO    MainThread:37814 [wandb_run.py:_redirect():1530] Redirecting console.
2021-10-06 23:43:49,610 INFO    MainThread:37814 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-06 23:43:49,610 INFO    MainThread:37814 [wandb_init.py:init():587] run started, returning control to user process
2021-10-06 23:43:49,677 INFO    MainThread:37814 [wandb_watch.py:watch():43] Watching
2021-10-06 23:43:49,701 INFO    MainThread:37814 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 1
2021-10-06 23:43:49,701 INFO    MainThread:37814 [wandb_run.py:_restore():1593] restore
2021-10-06 23:43:52,140 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:43:52,317 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 689
}

2021-10-06 23:43:52,486 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 3345
}

2021-10-06 23:43:52,592 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 5912
}

2021-10-06 23:43:52,698 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 689
  total_bytes: 5912
}

2021-10-06 23:43:52,802 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}

2021-10-06 23:43:52,917 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}

2021-10-06 23:43:53,021 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}

2021-10-06 23:43:53,124 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}

2021-10-06 23:43:53,286 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}

2021-10-06 23:43:53,608 INFO    MainThread:37814 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5912
  total_bytes: 5912
}
local_info {
}

2021-10-06 23:43:54,731 INFO    MainThread:37814 [wandb_run.py:_append_files():2009] logging synced files
